src/runtime/server/api/search.ts

```diff
@@ -0,0 +1,17 @@
+import { defineEventHandler } from 'h3'
+import { serverSearchContent, splitPageIntoSections } from '../search'
+import { useRuntimeConfig } from '#imports'
+
+export default defineEventHandler(async (event) => {
+  const runtimeConfig = useRuntimeConfig()
+  const { ignoredTags, ignoreQuery } = runtimeConfig.public.content.search
+
+  const files = await serverSearchContent(event, ignoreQuery)
```

Comment:
I think changing `ignoreQuery` to `filterQuery` will be simpler and more understandable for users.
We can have our default value to filter drafts, partials and ignored contents.